drizzle.config.ts

```
@@ -1,13 +1,13 @@
 export default defineConfig({
-  schema: "./src/database/schemas",
-  out: "./src/database/migrations",
-  dialect: "postgresql",
-  dbCredentials: {
-    url: process.env.DATABASE_URL!,
-  },
+	schema: "./src/database/schemas",
+	out: "./src/database/migrations",
+	dialect: "postgresql",
+	dbCredentials: {
+		url: process.env.DATABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)